.changeset/two-pumas-doubt.md

```diff
@@ -0,0 +1,11 @@
+---
+'@clerk/clerk-react': major
+'@clerk/chrome-extension': major
+---
+
+Drop deprecations. Migration steps:
```

Comment:
Isn't `useMagicLink` missing from that list?

Reply:
fixed!